process/subprocesses/ghsci.py

```diff
@@ -797,15 +793,13 @@ def _region_data_setup(
                     f"{region}.yml error: The 'data_dir' entry for {data} does not appear to have been defined.  This parameter is required for analysis of {region}, and is used to locate a required dataset cross-referenced in {region}.yml.  Please check the configured settings before proceeding.",
                 )
                 return None
-            if data_path is not None and not data_dictionary[
-                'data_dir'
-            ].startswith('Not required'):
+            if data_path is not None:
                 data_dictionary['data_dir'] = (
                     f"{data_path}/{data_dictionary['data_dir']}"
                 )
             return data_dictionary
         except Exception as e:
-            sys.exit(f'Data Check error with {data}: {e}')
+            sys.exit(e)

```

Comment:
I don't think a less informative error is helpful; I would not make this change.